### DISCUSSION TITLE
Application still stuck in "suppressed" state with no response for over a week

### DISCUSSION THREAD

**Discussion by axsuul** [amzn/selling-partner-api-models#3623] [General]
We have an urgent issue since our application has been in a "suppressed state" and new customers are unable to authorize with our Amazon Selling Partner application for over a week now. We have been following up on our case repeatedly with no response. We have also opened multiple cases but they just get closed and merged. Can anyone on the team please look into our case ID: 12961532901? 

Thanks

**Comment by axsuul** [comment]
Following up on this, thanks

**Comment by dipesh-emipro** [comment]
Hello,

We have encountered the same issue and have created a new case for App approval with the case number 12963804891.

**Comment by axsuul** [comment]
It's very frustrating. Still no response on any of our cases.

**Comment by rugved1991** [comment]
Hi,

The support team had received an influx of cases which caused delays in responding to your cases. The support cases have been addressed and the issue should now be resolved. If you have any further questions, feel free to respond on the support case or open a new case.
Have a great day!

Best,
Rugved
Solutions Architect, SP API
